src/Utils/Helpers.spec.ts

```diff
@@ -1,9 +1,101 @@
 import { Helpers } from './Helpers';
-import { should } from 'chai';
+import { should, expect } from 'chai';
+
+enum TestEnum {
+  TEST = 'test',
+  PROD = 'prod'
+}
 
 should();
 
 describe('Helpers', () => {
+  describe('getClusterUrls', () => {
+    it('should throw error when --cluster and --bus used', () => {
+      // arrange
+      const args = {
+        bus: 'amqps://localhost:5672',
+        cluster: 'localhost'
+      };
+      // act
+      const act = () => Helpers.getClusterUrls(args);
+      //assert
+      act.should.to.throw(
+        '--api and --bus options should not be used with --cluster option'
+      );
+    });
+  });
+
+  describe('getClusterUrls', () => {
+    it('should throw error when --cluster and --api used', () => {
+      // arrange
+      const args = {
+        api: 'http://localhost:8000',
+        cluster: 'localhost'
+      };
+      // act
+      const act = () => Helpers.getClusterUrls(args);
+      //assert
+      act.should.to.throw(
+        '--api and --bus options should not be used with --cluster option'
+      );
+    });
+  });
+
```

Comment:
Why use the multiple same `describe` for these tests?

```suggestion
  describe('getClusterUrls', () => {
    it('should throw error when --cluster and --bus used', () => {
      // arrange
      const args = {
        bus: 'amqps://localhost:5672',
        cluster: 'localhost'
      };
      // act
      const act = () => Helpers.getClusterUrls(args);
      //assert
      act.should.to.throw(
        '--api and --bus options should not be used with --cluster option'
      );
    });

    it('should throw error when --cluster and --api used', () => {
      // arrange
      const args = {
        api: 'http://localhost:8000',
        cluster: 'localhost'
      };
      // act
      const act = () => Helpers.getClusterUrls(args);
      //assert
      act.should.to.throw(
        '--api and --bus options should not be used with --cluster option'
      );
    });
  });

```